index.js

```diff
@@ -0,0 +1,2 @@
+require('babel-register');
```

Comment:
babel-register makes all subsequent 'requires' after `require('babel-register')` to be piped through babel, converting the code to es5. The `serve` task now starts the output of the piping by starting `index.js`, instead of the original `app.js`